nixos/tests/sourcehut/srht-gen-oauth-tok.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, pkgs, lib, fetchFromSourcehut }:
+
+let
+  perl = pkgs.perl.withPackages (pps: [
+    pps.CryptSSLeay
+    pps.WWWMechanize
+    pps.XMLLibXML
+  ]);
+in
+stdenv.mkDerivation rec {
```

Comment:
This should be placed under normal packages where someone else could reuse it, maybe for their own testing suite.